test/test_resize.cpp

```diff
@@ -3115,4 +3115,50 @@ TEST_F(NVFuserTest, dynamicReshapeIssue1393) {
   testValidate(fusion, outputs, {t0, t1}, {ref}, __LINE__, __FILE__);
 }
 
+// Test that we can slice a trivially expanded tensor to size 1 then squeeze
+// See https://github.com/NVIDIA/Fuser/issues/963
```

Comment:
This test should've been added in #975, but I'm including it here since it showcases the simplification.